tests/NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests/Steps/AccessTokenSteps.cs

```diff
@@ -61,5 +61,44 @@ public void TheUserIsLoggedInWithRoleForOrganisation(string role, string organis
             var token = builder.BuildToken();
             _context[ScenarioContextKeys.AccessToken] = token;
         }
+
+        [Given(@"the user (.*) with id (.*)is logged in with the (Buyer|Authority|Readonly-Buyer) role for organisation (.*)")]
```

Comment:
As mentioned on the call I don't think we need this extra step which is duplicating all the claims information that is already in another step. On the call we agreed to change the initial data in the database & use the details of the currently logged in user.